polaris-react/src/components/Page/components/Header/components/Title/Title.tsx

```diff
@@ -33,7 +33,13 @@ export function Title({
     subtitle && styles.TitleWithSubtitle,
   );
 
-  const titleMarkup = title ? <h1 className={className}>{title}</h1> : null;
+  const titleMarkup = title ? (
+    <h1 className={className}>
+      <Text as="span" variant="headingLg" fontWeight="bold">
```

Comment:
Just noting that the `headingLg` variant has uses the `--p-font-letter-spacing-dense` as opposed to `normal` so and it's causing a diff in Chromatic but we don't have support for overriding letter spacing on Text so that's probably fine 👍